src/types.ts

```diff
@@ -123,3 +115,7 @@ export type Operation = {
   links: Links;
   meta: Meta;
 };
+
+export type OperationResponse = {
+  data: Resource | Resource[] | null;
```

Comment:
@Exelord I'm uncertain that we want to allow `null` as a value. I'd rather mark it as `undefined` with `data?`.

Reply:
According to JSONAPI doc, there are cases when we need to return `data: null` so it might be one of those.